src/Analyzers/Orchestration/OrchestrationAnalyzer.cs

```diff
@@ -10,10 +10,95 @@
 
 namespace Microsoft.DurableTask.Analyzers.Orchestration;
 
+/// <summary>
+/// Base class for visitors that analyze orchestrations.
```

Comment:
Can you add a longer description in this summary of how the `OrchestrationVisitor` class is used. A high level design description would be helpful for maintainability in the future. Thanks!